src/OpenColorIO/PathUtils.cpp

```diff
@@ -57,6 +56,9 @@ std::string ComputeHash(const std::string & filename)
     return "";
 }
 
+//Holding default hash function
```

Comment:
As mentioned that should be a sentence:
// The global variable holds the hash function to use. It could be changed using SetComputeHashFunction() to customize the implementation.

Reply:
Updated it. 👍🏻